src/area-chart/__integ__/area-chart.test.ts

```diff
@@ -488,7 +491,23 @@ describe('Controlled', () => {
     })
   );
 });
-
+describe.only('Loading state', () => {
+  test(
+    'height of the chart is displayed correctly after changing loading state to finished',
+    setupTest('#/light/area-chart/loading', 'chart', async page => {
+      await page.setWindowSize({ width: 500, height: 800 });
+      // the loading is about 1000ms in the loading page and then the
+      // statusType becomes 'finished'. This just waits for it.
+      await page.waitForJsTimers(2000);
+
+      // we only need to validate one of the series here, looping over all
+      // of the series will cause a failure since we might have one series with height 0 (threshold one)
+      const seriesBox = await page.getBoundingBox(computedDomainChartWrapper.findSeries().get(1).toSelector());
+      expect(seriesBox.height).toBeGreaterThan(100);
```

Comment:
Minor: The area chart has height as 200, can we make it check it is greater than 200?